com.unity.formats.fbx/Documentation~/export-timeline-clip.md

```diff
@@ -0,0 +1,25 @@
+# Export a Timeline animation clip to FBX
+
+Export any Timeline clip containing animation to an FBX file.
+
+>**Note:** This scenario requires to have the [Timeline](https://docs.unity3d.com/Packages/com.unity.timeline@latest) package installed in addition to the FBX Exporter package.
+
+To export an animation clip:
+
+1. In the Timeline window, right-click on the clip to export, and then select **Export Clip To FBX**.
```

Comment:
another suggestion, I think it would be nice to have an image of what it looks like in the timeline